src/Idris/IDEMode/CaseSplit.idr

```diff
@@ -229,9 +240,15 @@ dropLast updChars with (snocList updChars)
 rtrim : String -> String
 rtrim = reverse . ltrim . reverse
 
+-- remove last paren and everything after it
+dropLastParen : SnocList Char -> SnocList Char
+dropLastParen Lin = Lin
+dropLastParen (cs :< ')') = cs
+dropLastParen (cs :< c) = dropLastParen cs :< c
```

Comment:
Isn't this preserving everything after the last parens?
In which case you need to change the comment